src/components/ChatroomsList.tsx

```diff
@@ -0,0 +1,155 @@
+import Link from "next/link";
+import RoomCodeButton from "./RoomCodeButton";
+import CopyUrlButton from "./CopyUrlButton";
+import DeleteRoomButton from "./DeleteRoomButton";
+
+interface Chatroom {
+  id: string;
+  roomUrl?: string;
+  title: string;
+  createdAt: string;
+  _count: {
+    messages: number;
+  };
+}
+
+interface ChatroomsListProps {
+  chatrooms: Chatroom[];
+  isLoading: boolean;
+  handleCopyRoomUrl: (room: Chatroom) => void;
+  handleCopyRoomId: (room: Chatroom) => void;
+  handleDeleteRoom: (room: Chatroom) => void;
+}
+
+export default function ChatroomsList({
+  chatrooms,
+  isLoading,
+  handleCopyRoomUrl,
+  handleCopyRoomId,
+  handleDeleteRoom,
+}: ChatroomsListProps) {
+  return (
+    <div className="bg-card rounded-lg shadow-sm border border-border mb-6">
+      <div className="p-4 border-b border-gray-200 dark:border-gray-700">
+        <h2 className="text-lg font-semibold text-gray-900 dark:text-white">
+          Active Chatrooms
+        </h2>
+      </div>
+      <div className="p-4">
+        {isLoading ? (
+          <div className="text-center py-4">
+            <div className="text-gray-600 dark:text-gray-400">
+              Loading chatrooms...
+            </div>
+          </div>
```

Comment:
```suggestion
          <span className="text-center py-4 text-gray-600 dark:text-gray-400">
            Loading chatrooms...
          </span>
```
Can we remove this div also? Its divception in there, in fact it probably should be a span seeing as it contains only text